docs/source/multidc/multidc.md

```diff
@@ -561,3 +561,40 @@ UN  172.16.39.209  336 KB     256          ?       7c30ea55-7a4f-4d93-86f7-c8817
 UN  172.16.25.18   759 KB     256          ?       665dde7e-e420-4db3-8c54-ca71efd39b2e  a
 UN  172.16.87.27   503 KB     256          ?       c19c89cb-e24c-4062-9df4-2aa90ab29a99  c
 ```
+
+## Scylla Manager
+
+Integration with Scylla Manager requires having it installed in just one DC.
```

Comment:
```suggestion
To integrate a multi-datacenter ScyllaDB clustere with Scylla Manager, you must deploy the Scylla Manager in only one datacenter.
```

Reply:
unless it can be deployed in both? I think the wording needs to be specific if it must be installed in one and only one

Reply:
done